react-quill/README.md

```diff
@@ -0,0 +1,27 @@
+# Using react-quill (WYSIWYG) in Remix application
+
+In this example we are implementing a WYSIWYG editor with Quill. 
+
+We have used react-quill as our library of choice to implement this in action.
+
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+<!-- TODO: update this link to the path for your example: -->
+
```

Comment:
```suggestion
```

Reply:
https://codesandbox.io/s/react-quill-ff1tmg

I suppose this thing should be a fork inside of the code sandbox. 

But I don't know how it will be a fork from __example (But scaffolds an empty no-dependency project every time)

Reply:
Please remove this comment